docs/modules/ROOT/pages/guides/v4-migration/index.adoc

```diff
@@ -292,3 +292,47 @@ type Query {
   moviesByTitle(phrase: String!, where: MovieFulltextWhere, sort: [MovieFulltextSort!], limit: Int, offset: Int): [MovieFulltextResult!]!
 }
 ----
+
+=== `@cypher` changes
+The default behaviour of the `@cypher` directive regarding the translation will change: Instead of using https://neo4j.com/labs/apoc/4.0/overview/apoc.cypher/apoc.cypher.runFirstColumnMany/[apoc.cypher.runFirstColumnMany] it will directly wrap the query within a `CALL { }` subquery. This behvaiour has proven to be much more performant for the same queries, however, it may lead to unexpected changes, mainly when using Neo4j 5.x, where the subqueries need to be _aliased_.
+
+For example:
+
+The graphql query:
+[source, graphql, indent=0]
+----
+type query {
+    test: String! @cypher(statement: "RETURN 'hello'")
+}
+----
+
+Would get translated to:
+[source,cypher, indent=0]
+----
+CALL {
+    RETURN 'hello'
+}
+WITH 'hello' AS this
+RETURN this
+----
+
+Which is invalid in Neo4j 5.x.
+
+To fix it we just need to ensure the `RETURN` elements are aliased:
+[source, graphql, indent=0]
+----
+type query {
+    test: String! @cypher(statement: "RETURN 'hello' as result", experimental: true)
+}
+----
+
+This will be a breaking change, but this new behaviour can be used, as an experimental option with the `experimental` flag in the `@cypher` directive:
```

Comment:
Is it worth making clear here that the experimental flag is just to use the feature early and won't actually be available/required in v4 seeing as this is the migration guide?